src/i965_decoder_utils.c

```diff
@@ -23,7 +23,17 @@
 
 #include "sysdeps.h"
 #include <limits.h>
-#include <alloca.h>
+#ifdef STDC_HEADERS
+# include <stdlib.h>
```

Comment:
sysdeps.h has already included stdlib.h, it would be better to move this check to sysdeps.h